code/__SPLURTCODE/DEFINES/is_helpers.dm

```diff
@@ -6,3 +6,5 @@
 #define iswendigo(A) (istype(A, /mob/living/carbon/wendigo))
 
 #define iscloudturf(A) (istype(A, /turf/open/chasm/cloud))
+
+#define isarachnid(A) (is_species(A, /datum/species/arachnid) || HAS_TRAIT(A, TRAIT_ARACHNID))
```

Comment:
I would rather if instead you kept the old definition and just named the new one something else, or directly edit the nonmodular one in this case, depending on what you need it for

Reply:
I can't edit the nonmodular one because it relies on something defined in `_SPLURTCODE`, not can I just make a new definition because then I'd have to change *everything using* `isarachnid` to use `isarachnidnew`

Reply:
... Ooon second thought, scratch that first part, seems I can do exactly that. I'll push my commit out in just a second.